### ISSUE TITLE
RAND_load_file changed API and manpage is wrong

### ISSUE THREAD

**Issue by mcepl** [openssl/openssl#4536]
Not sure, when it happened, but ``RAND_load_file(3)`` now returns ``-1`` when the file does not exist. It used to return ``0`` (as number of returned bytes). Also, the manpage says:
```
RETURN VALUES
       RAND_load_file() returns the number of bytes read.
```
which is not true anymore.

**Comment by richsalz** [comment]
IT looks like the change was in 1.1.0.  The manpage should be updated.

**Comment by mattcaswell** [comment]
Why are we closing this if there a man page bug?

**Comment by richsalz** [comment]
I hit the wrong button; thanks.

**Comment by richsalz** [comment]
Doc  fix for 1.1.0 and master pushed.  Closing.
